test/unit/specs/components/Identity/getIdentity/createGetIdentity.spec.js

```diff
@@ -42,8 +42,20 @@ describe("Identity::createGetIdentity", () => {
   });
 
   it("each getIdentity call should create new payloads and requests", () => {
-    const payload1 = { type: "payload1" };
-    const payload2 = { type: "payload2" };
+    const payload1 = jasmine.createSpyObj(
+      "requestPayload",
+      ["mergeConfigOverride"],
+      {
+        type: "payload1"
+      }
+    );
+    const payload2 = jasmine.createSpyObj(
+      "requestPayload",
+      ["mergeConfigOverride"],
+      {
+        type: "payload2"
+      }
+    );
     const request1 = { type: "request1" };
     const request2 = { type: "request2" };
     createIdentityRequestPayload.and.returnValues(payload1, payload2);
```

Comment:
No verifications that mergeConfigOverride is called?

Reply:
There are some, further down in the test. I didn't include them in this test case because it's not about config overrides and because [`mergeConfigOverrides()` is always called](https://github.com/adobe/alloy/blob/config-overrides/src/components/Identity/getIdentity/createGetIdentity.js#L23), even when there are no config overrides.